### PR TITLE
Added email + discord integration so we can collect contact emails

### PR DESCRIPTION
**Pull request by alexvobo**
This allows us to collect emails and push them to a discord channel via webhook.

### CODE REVIEW

File: pages/index.js [hunk 1]
```diff
@@ -1,7 +1,33 @@
...
+import Head from "next/head";
+import styles from "../styles/Home.module.css";
+import validator from "validator";
+import axios from "axios";
 
 export default function Home() {
+  const inputRef = useRef(null);
+  const [email, setEmail] = useState("");
+  const [emailValid, setEmailValid] = useState(false);
+  const [placeHolder, setPlaceHolder] = useState("Email Address");
```
Comment (airdropross) [inline pages/index.js]:
```suggestion
  const [placeHolder, setPlaceHolder] = useState(PLACEHOLDER);
```

File: pages/index.js [hunk 2]
```diff
@@ -1,7 +1,33 @@
-import Head from 'next/head'
-import styles from '../styles/Home.module.css'
+import { useState, useRef } from "react";
+
+import Head from "next/head";
+import styles from "../styles/Home.module.css";
+import validator from "validator";
+import axios from "axios";
```
Comment (airdropross) [inline pages/index.js:7]:
```suggestion

const PLACEHOLDER = "Drop your email for info";
```

File: pages/index.js [hunk 3]
```diff
@@ -1,7 +1,33 @@
...
+    }
+    setEmail("");
+    setEmailValid(false);
+    setPlaceHolder("Speak soon!");
+    inputRef.current.style.animation = "none";
+  };
+
+  const validateInput = (e) => {
+    var userInput = e.target.value;
+    setPlaceHolder("Email Address");
```
Comment (airdropross) [inline pages/index.js]:
```suggestion
    setPlaceHolder(PLACEHOLDER);
```

**Review by airdropross — APPROVED** [review]

File: pages/index.js (hunk 2 above)
Comment (alexvobo) [reply pages/index.js:7]: implemented

File: pages/index.js (hunk 1 above)
Comment (alexvobo) [reply pages/index.js]: implemented

File: pages/index.js (hunk 3 above)
Comment (alexvobo) [reply pages/index.js]: Implemented

**Pull request merged**